cdap-docs/reference-manual/source/http-restful-api/security.rst

```diff
@@ -289,27 +310,28 @@ with the secure key returned as text, passed in the response body.
    * - Status Codes
      - Description
    * - ``200 OK``
-     - A key was successfully retrieved
+     - The secure key was successfully retrieved
    * - ``404 NOT FOUND``
-     - The namespace specified in the request does not exist, or secure key by the specified name does not exist in the
-       specified namespace
+     - The namespace specified in the request does not exist or the secure key with that
+       name does not exist in that namespace
 
 
 Retrieve the Metadata for a Secure Key
 --------------------------------------
-
-You can retrieve only the metadata for a key from secure storage by making an HTTP GET request to the URL::
+You can retrieve just the metadata for a secure key from secure storage by making an HTTP GET request to the URL::
```

Comment:
Fixed, though I think in this context, "metadata" is correct and no longer confusing.